From c0a7ac13565248efef3bee88e2e2481b994bb823 Mon Sep 17 00:00:00 2001
From: Michael Catanzaro <mcatanzaro@igalia.com>
Date: Sun, 10 Jan 2016 10:59:47 -0600
Subject: [PATCH 07/12] lib: fix misleading debug message

This debug message unnecessarily implies that the accounts daemon is
not running. There are other reasons why loading a user could fail;
e.g. if a username simply does not exist.

https://bugs.freedesktop.org/show_bug.cgi?id=93655
---
 src/libaccountsservice/act-user-manager.c | 6 ++++--
 1 file changed, 4 insertions(+), 2 deletions(-)

diff --git a/src/libaccountsservice/act-user-manager.c b/src/libaccountsservice/act-user-manager.c
index 0907514..914a1c9 100644
--- a/src/libaccountsservice/act-user-manager.c
+++ b/src/libaccountsservice/act-user-manager.c
@@ -2212,9 +2212,11 @@ static void
 give_up (ActUserManager                 *manager,
          ActUserManagerFetchUserRequest *request)
 {
+        if (request->type == ACT_USER_MANAGER_FETCH_USER_FROM_USERNAME_REQUEST)
+                g_debug ("ActUserManager: failed to load user %s", request->username);
+        else
+                g_debug ("ActUserManager: failed to load user %lu", (gulong) request->uid);
 
-        g_debug ("ActUserManager: account service unavailable, "
-                 "giving up");
         request->state = ACT_USER_MANAGER_GET_USER_STATE_UNFETCHED;
 
         if (request->user)
-- 
1.9.5

